package.json

```diff
@@ -65,12 +65,12 @@
     "clean": "npm run clean:dist && npm run clean:example && npm run clean:scss",
     "clean:dist": "del dist",
     "clean:example": "del docs && del .nuxt",
-    "clean:scss": "del src/scss",
+    "clean:scss": "del-cli src/scss",
```

Comment:
`del-cli` has to be added as `devDep` and also update the command on `clean:dist` and `clean:example`.

Reply:
`del-cli` is already in `devDependency`.